src/theme/semantic-tokens.ts

```diff
@@ -10,6 +10,10 @@ export const semanticTokens: SemanticTokens = {
       default: 'white',
       _dark: 'gray.800',
     },
+    toolTipBg: {
+      default: 'whiteAlpha.900',
```

Comment:
The color for this toolTipBg already exists in line 147-150 and the dark is gray 700 from the design